applications/fluid_structure_interaction/perpendicular_flap/application.h

```diff
@@ -619,7 +620,7 @@ class Application : public StructureFSI::ApplicationBase<dim, Number>
     param.problem_type         = ProblemType::Unsteady;
     param.body_force           = false;
     param.pull_back_body_force = false;
-    param.large_deformation    = true;
+    param.large_deformation    = false;
```

Comment:
this changes the mathematical model (linear elasticity is now used). Is this intended?

Reply:
It's actually not intended as part of this workpackage (it has nothing to do with time), but I would like to change it for consistency reasons as the corresponding tutorial also relies on linear elasticity models on the structural side.